src/solver/clause.rs

```diff
@@ -321,14 +322,15 @@ impl Clause {
 /// variable are grouped together in a linked list, so it becomes easy to notify
 /// them all.
 #[derive(Clone)]
-pub(crate) struct ClauseState {
-    // The ids of the literals this clause is watching
-    pub watched_literals: [Literal; 2],
+pub(crate) struct ClauseWatches {
```

Comment:
Wouldn't `WatchedLiterals` make more sense, also reading the description?

Reply:
Yes much better, changed it!

Reply:
Glad to be of service :)